apps/anmtool/anmtool.cpp

```diff
@@ -133,20 +133,20 @@ bool parseOptions(int argc, char** argv, Arguments& args, int& return_code)
 	cxxopts::Options options("anmtool", "Inspect and extract files from LionHead ANM files.");
 
 	// clang-format off
-	options.add_options()
-		("h,help", "Display this help message.")
-		("subcommand", "Subcommand.", cxxopts::value<std::string>())
-	;
-	options.positional_help("[read|write] [OPTION...]");
-	options.add_options()
-		("H,header", "Print Header Contents.", cxxopts::value<std::vector<std::string>>())
-		("l,list-keyframes", "List Keyframes.", cxxopts::value<std::vector<std::string>>())
-		("k,keyframe-content", "View Keyframe Contents", cxxopts::value<std::vector<std::string>>())
-	;
-	options.add_options("write from and to glTF format")
-		("o,output", "Output file (required).", cxxopts::value<std::string>())
-		("i,input-mesh", "Input file (required).", cxxopts::value<std::string>())
-	;
+  options.add_options()
+    ("h,help", "Display this help message.")
+    ("subcommand", "Subcommand.", cxxopts::value<std::string>())
+    ;
+  options.positional_help("[read|write] [OPTION...]");
+  options.add_options()
+    ("H,header", "Print Header Contents.", cxxopts::value<std::vector<std::filesystem::path>>())
+    ("l,list-keyframes", "List Keyframes.", cxxopts::value<std::vector<std::filesystem::path>>())
+    ("k,keyframe-content", "View Keyframe Contents", cxxopts::value<std::vector<std::filesystem::path>>())
+    ;
+  options.add_options("write from and to glTF format")
+    ("o,output", "Output file (required).", cxxopts::value<std::filesystem::path>())
+    ("i,input-mesh", "Input file (required).", cxxopts::value<std::filesystem::path>())
+    ;
```

Comment:
Hard to tell on Github but are these now tabs?

Reply:
No, these are still spaces.
I didn't pay attention to that, but it looks like since this section has clang-format disabled, my editor automatically used its default indentation configuration, that is "two spaces"-indentation.